$common/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/python3
```

Comment:
I guess this file shouldn't be present here.

Reply:
Indeed I changed that now.